index.html

```diff
@@ -22,17 +22,21 @@ <h1 class="name">Anne Hölscher</h1>
     </p>
```

Comment:
To be more semantic I recommend you to divide your code in `header`, `main` and `footer` content.